embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java

```diff
@@ -177,11 +178,12 @@ protected ClientConfiguration getClientConfiguration(PluginTask task)
     {
         ClientConfiguration clientConfig = new ClientConfiguration();
 
+        /** PLT-9886: disable built-in retry*/
         //clientConfig.setProtocol(Protocol.HTTP);
-        clientConfig.setMaxConnections(50); // SDK default: 50
-        clientConfig.setMaxErrorRetry(3); // SDK default: 3
-        clientConfig.setSocketTimeout(8 * 60 * 1000); // SDK default: 50*1000
-
+//        clientConfig.setMaxConnections(50); // SDK default: 50
+//        clientConfig.setMaxErrorRetry(3); // SDK default: 3
+//        clientConfig.setSocketTimeout(8 * 60 * 1000); // SDK default: 50*1000
+        clientConfig.withRetryPolicy(PredefinedRetryPolicies.NO_RETRY_POLICY);
```

Comment:
Please actually test this to be sure setting retry policy really has the expected effect, and it it doesn't accidentally be overridden somewhere, my tests on seems yield an unexpected result here.
Also, I would prefer `setRetryPolicy` over `withRetryPolicy`. `with...` is kind of ambiguous without chaining calls, sometimes it indicates an immutable method.

Reply:
I debugged and it worked with RetryPolicy.